src/app/rpmostree-builtin-status.cxx

```diff
@@ -631,14 +632,19 @@ print_one_deployment (RPMOSTreeSysroot *sysroot_proxy,
                 g_print ("%s", custom_origin_url);
               }
             else
-              g_print ("%s", canonrefspec);
+              g_print ("%s", origin_refspec);
           }
           break;
         case RPMOSTREE_REFSPEC_TYPE_OSTREE:
           {
-            g_print ("%s", canonrefspec);
+            g_print ("ostree-refspec:  %s", origin_refspec);
```

Comment:
I think I'd prefer to not print these prefixes; it's usually going to be obvious to a human which is actually in use.